example/server.js

```diff
@@ -2,22 +2,44 @@
 
 var Hapi = require('hapi');
 var Hoek = require('hoek');
-var tags = require('../lib/index.js');
+var fs = require('fs');
 
 var server = new Hapi.Server();
+var tagsData = JSON.parse(fs.readFileSync('./example/tags.json').toString());
+var tags = require('../lib/index.js');
+var tagsPool = require('./pg.js');
```

Comment:
See my comment on the pg.js file. I think you can just create the pool inside init:
```
var pool = new pg.Pool(pgConfig);
```